src/mappers/jsonMapper.ts

```diff
@@ -1,11 +1,9 @@
-import { ValueMapper } from '../abstracts/ValueMapper';
-
 type JsonMapper = <TJsonResult>(value: string) => TJsonResult | null;
 
 export const jsonMapper: JsonMapper = <TJsonResult extends any>(value: string): TJsonResult | null => {
     try {
-        return JSON.parse(value)
+        return JSON.parse(value);
```

Comment:
Thank you for that. I always forget to set semicolon, because they are deactivated in the other projects I work with :-)

Reply:
it was done automatically by commend `yarn format && yarn lint:fix` :)